docs/_documentations/troubleshooting.md

```diff
@@ -504,7 +504,25 @@ Issue link: https://github.com/eclipse/codewind-docs/issues/64
 ## Node.js and Swift templates remain in the starting state
 The templates `Appsody Node.js template` and `Appsody Swift template` remain in the starting state by default because these templates do not have a server implemented, and therefore, its status cannot be detected. These templates do not have a server and are intended to help you implement your own server.
```

Comment:
I think there might be an unnecessary space before the `The` at the beginning of this first sentence.

Reply:
I don't see the space in the editor.  In any case I didn't touch this sentence.

Reply:
Cool, thanks, I wasn't sure if there were, thought I'd point it out in case.